### ISSUE TITLE
Unable to handle click on overlay

### ISSUE THREAD

**Issue by Kreozot** [jasongaare/react-native-walkthrough#18]
If user wants to stop walkthrough by clicking on overlay, there is no way for this. Or do I miss someting?

**Comment by jasongaare** [comment]
Checkout the `onClose` prop from https://github.com/jasongaare/react-native-walkthrough you can pass to `tooltipProps`

Does this do what you're wanting?

**Comment by Kreozot** [comment]
No. Because it fires independent of how you close a tooltip (you can click "next" for example)

**Comment by TweetyBoop1990** [comment]
> No. Because it fires independent of how you close a tooltip (you can click "next" for example)

Did you every figure out how to do this?
